docker/px4-dev/Dockerfile_simulation

```diff
@@ -21,12 +21,16 @@ RUN wget --quiet http://packages.osrfoundation.org/gazebo.key -O - | apt-key add
 		python-numpy \
 	&& apt-get -y autoremove \
 	&& apt-get clean autoclean \
-	# px4tools, pymavlink
-	&& pip install px4tools pymavlink \
-	# dronekit
+	# px4tools
+	&& pip install px4tools \
```

Comment:
Or maybe we should leave them all at the latest versions as a way of finding issues? Our usage of docker images is always tagged.

I hope to be using this soon for mission tests.